frontend/src/scenes/dashboard/Dashboards.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { dashboardsModel } from '~/models/dashboardsModel'
+import { Button, Spin } from 'antd'
+import { dashboardsLogic } from 'scenes/dashboard/dashboardsLogic'
+import { Link } from 'lib/components/Link'
+import { PlusOutlined } from '@ant-design/icons'
+import { Table } from 'antd'
+
+export default function Dashboards() {
+    const { dashboardsLoading } = useValues(dashboardsModel)
+    const { dashboards } = useValues(dashboardsLogic)
+    const { addNewDashboard } = useActions(dashboardsLogic)
+
+    return (
+        <div>
+            <div style={{ marginBottom: 20 }}>
+                <Button onClick={addNewDashboard} style={{ float: 'right' }}>
+                    <PlusOutlined style={{ verticalAlign: 'baseline' }} />
+                    New Dashboard
+                </Button>
+                <h1>Dashboards</h1>
+            </div>
+
+            {dashboardsLoading ? (
+                <Spin />
+            ) : dashboards.length > 0 ? (
+                <Table
+                    dataSource={dashboards}
+                    rowKey="id"
+                    size="small"
+                    pagination={{ pageSize: 100, hideOnSinglePage: true }}
+                >
+                    <Table.Column
```

Comment:
Might be worth adding the delete button here too

Reply:
![image](https://user-images.githubusercontent.com/53387/81583812-2efcc280-93b2-11ea-8eb1-894901283a5e.png)